android/assets/jsons/translations/Portuguese.properties

```diff
@@ -834,26 +834,26 @@ New map = Novo mapa
 Empty = Vazio
 Language = Língua
  # Requires translation!
-Terrains & Resources = 
+Terrains & Resources = Terrenos & Recursos
  # Requires translation!
-Improvements = 
+Improvements = Melhorias
 Clear current map = Limpar mapa atual
 Save map = Salvar mapa
 Download map = Baixar mapa
 Loading... = Carregando...
 Filter: = Filtrar:
 Exit map editor = Sair do editor de mapa
  # Requires translation!
-[nation] starting location = 
+[nation] starting location = Localização Inicial do(a) [nation]
  # Requires translation!
-Clear terrain features = 
+Clear terrain features = Limpar características de terreno
  # Requires translation!
-Clear improvements = 
+Clear improvements = Limpar melhorias
 Clear resource = Limpar recursos
 Requires = Requer
 Menu = Menu
  # Requires translation!
-Brush Size = 
+Brush Size = Tamanho de Arbusto
```

Comment:
Sounds like a mistake like: arbusto = bush / b**r**ush = pincel ou algo parecido

Reply:
Correct. I thought this was some terrain editing mod feature :(
it should be 'Pincel' instead of 'Arbusto'